### ISSUE TITLE
Updates

### ISSUE THREAD

**Issue by cossssmin** [posthtml/posthtml-postcss#55]
I want to start work on updating this plugin, here's what I'm planning:

- `v0.6.0` -> CJS, bring up to update in general (dependencies, CI, tests with Vitest etc.) ✅
	- BREAKING: require Node.js ~12+~ ~14+~ 16+
- `v1.0.0` -> ESM-only
	- BREAKING: require Node.js 18+

I will need to be added to NPM collaborators to be able to publish releases @voischev @Scrum

**Comment by Scrum** [comment]
@cossssmin invite sending

**Comment by cossssmin** [comment]
Thanks Ivan!

**Comment by cossssmin** [comment]
@Scrum `posthtml-postcss` is not added to the PostHTML org on NPM, so I can't publish a new release.

https://www.npmjs.com/org/posthtml

```sh
? Select SemVer increment or specify new version minor  0.6.0

  > Prerequisite check
    √ Ping npm registry
    √ Check npm version
    × Verify user is authenticated
      → You do not have write permissions required to publish this package.
      Check git version
      Check git remote
      Validate version
      Check for pre-release version
      Check git tag existence
    Git
    Installing dependencies using npm
    Running tests using npm
    Bumping version using npm
    Publishing package using npm
    Pushing tags
    Creating release draft on GitHub

✖ Error: You do not have write permissions required to publish this package.
    at Task.task (file:///F:/DEV/open-source/posthtml/posthtml-postcss/node_modules/np/source/prerequisite-tasks.js:47:12)
    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)

Aborted!
```

**Comment by cossssmin** [comment]
Published [v1.0.0](https://github.com/posthtml/posthtml-postcss/releases/tag/v1.0.0), closing issue as resolved 👍
